api/pytest.ini

```diff
@@ -5,5 +5,21 @@ pythonpath = .
 
 filterwarnings=
   ignore:Unverified HTTPS request
```

Comment:
General nit: do you mind moving the comments to be *above/before* the code they're commenting, rather than on the line after?

Reply:
ok